Amplication CLI build failed on compatibility with cross-fetch

based on #1278 

I am following installation instructions from scratch. when running npm run bootstrap, I am getting the following error :

src/configured-command.ts:50:7 - error TS2322: Type '(input: import("C://My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").RequestInfo, init?: import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").RequestInit | undefined) => Promise<...>' is not assignable to type '(input: RequestInfo, init?: RequestInit | undefined) => Promise'.
Types of parameters 'input' and 'input' are incompatible.
Type 'RequestInfo' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").RequestInfo'.
Type 'Request' is not assignable to type 'RequestInfo'.
Type 'Request' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").Request'.
Types of property 'body' are incompatible.
Type 'ReadableStream | null' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").ReadableStream | null'.
Type 'ReadableStream' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").ReadableStream'.
Types of property 'getReader' are incompatible.
Type '{ (options: { mode: "byob"; }): ReadableStreamBYOBReader; (): ReadableStreamDefaultReader; }' is not assignable to type '() => ReadableStreamDefaultReader'.

--- a/packages/amplication-cli/package.json
+++ b/packages/amplication-cli/package.json
@@ -51,7 +51,7 @@
     "@types/pluralize": "^0.0.29",
     "@types/url-join": "^4.0.0",
     "cli-ux": "^5.5.1",
-    "cross-fetch": "^3.0.6",
+    "cross-fetch": "3.0.6",
     "graphql": "^15.5.0",
     "pascal-case": "^3.1.2",
     "pluralize": "^8.0.0",

--- a/packages/amplication-data-service-generator/package.json
+++ b/packages/amplication-data-service-generator/package.json
@@ -108,7 +108,7 @@
     "@typescript-eslint/parser": "^4.6.1",
     "babel-eslint": "^10.0.0",
     "base-64": "^0.1.0",
-    "cross-fetch": "^3.0.6",
+    "cross-fetch": "3.0.6",
     "docker-compose": "^0.23.5",
     "eslint": "^7.17.0",
     "eslint-config-react-app": "^6.0.0",

--- a/packages/amplication-design-system/package.json
+++ b/packages/amplication-design-system/package.json
@@ -29,7 +29,8 @@
     "react-animate-height": "^2.0.23",
     "react-device-detect": "^1.14.0",
     "react-select": "^3.1.1",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "formik": "^2.2.5"
   },
   "devDependencies": {
     "@storybook/addon-actions": "^6.1.9",
@@ -69,7 +70,6 @@
   },
   "peerDependencies": {
     "@rmwc/provider": "^6.1.4",
-    "formik": "^2.2.5",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "styled-components": "^5.1.1"

